OWASP/SecureTea-Project
# Aws ses

## Status
**READY**

## Description
The following PR introduces AWS SES services in the secureTea project. It sends email notifications regarding security threats observed.

## Related PRs
None

## Todos
- [ ] Documentation in README required


## Deploy Notes
Run pip install -r requirements.txt again

## Steps to Test or Reproduce
1. pip install -r requirements.txt
2. Setup your aws ses credentials. For that you will need a Amazon Developer's account. If account is already created proceed to step 6
3. Setup a amazon developers account here: https://aws.amazon.com/
4. Setup SES credentials using first 12 steps here: https://www.formget.com/setup-amazon-ses-account/
5. Add your email id to verified email ids. For that refer to SES email ids section of your developer account
6. Enter AWS credentials when asked for
7. Test the service by moving the mouse to detect an anomaly.

Refer to [AWS_SES.gif](https://i.ibb.co/PYpd6MV/AWS-SES.gif) for the flow!

In case if you don't want to create an account, contact me at majmundarkushal9@gmail.com. I will provide you with my ses credentials and will verify your email and will ask you too reproduce some other steps.
```sh
git pull --prune
git checkout <feature-branch>
```

1. 

## Impacted Areas in Application
List general components of the application that this PR will affect:

args,
libs/notifs
requirements
core
*
